x-pack/docs/en/rest-api/security/create-users.asciidoc

```diff
@@ -26,12 +26,12 @@ to as _native users_.
 [[security-api-put-user-desc]]
 ==== {api-description-title}
 
-When updating a user, you can update everything but its `username` and `password`. 
-To change a user's password, use the 
-<<security-api-change-password, change password API>>.
+A `password` is required for adding a new user but is optional when updating an
+existing user. To change a user's password,
```

Comment:
This reads a little strangely to me now - it says _you can optionally change the password here_ but also _you should use this other API_.
 
```suggestion
existing user. To change a user's password without updating any other fields,
```